projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -0,0 +1,20 @@
+<cx-carousel
+  [items]="items$ | async"
+  [title]="title$ | async"
+  [template]="merchandisingCarouselItem"
```

Comment:
we should keep the naming convention consistent through carousels, therefore I believe it should be named carouselItem

Reply:
Done